{
  Boolean b=Math.random() < 0 ? Boolean.TRUE : null;
  try {
    assertEquals(null,b.booleanValue());
    fail("Should have thrown exception");
  }
 catch (  Exception e) {
  }
  try {
    boolean bb=b;
    fail("Should have thrown exception" + bb);
  }
 catch (  Exception e) {
  }
}
